language-server/modules/langserver-core/src/test/resources/codeaction/import-module/source/project/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "lstest"
+name = "project"
+version = "0.1.0"
+distribution = "2201.1.1"
+
+[build-options]
```

Comment:
Shall we exclude this build option. This might cause compilation errors as we don't have the `observability pack`  in the lang